array-loops.js

```diff
@@ -32,4 +55,17 @@ const figures = [4, 6, 12, 34, 1];
 // Wanneer je de volledige array logt zie je alle nieuwe waardes
 // console.log(figures) geeft: [ 7, 9, 15, 37, 4 ]
 
+const add_number = (numberToAdd, array) => {
```

Comment:
@NWaziri: zie opmerkingen regel #14

(uitwerking in opdrachten)

![image](https://user-images.githubusercontent.com/74968337/105503667-2b526700-5cc7-11eb-9a94-5d3cc6fe4d93.png)